app/billing/rest.py

```diff
@@ -192,3 +204,17 @@ def update_free_sms_fragment_limit_data(service_id, free_sms_fragment_limit, fin
             free_sms_fragment_limit,
             financial_year_start
         )
+
+
+def serialize_ft_billing(data):
```

Comment:
Is this the best place for this? I'd usually but it in a business model but we don't have one?
The other place would be the DAO but that also feels wrong?

Reply:
I moved it to billing_schemas.py